Formula/astro@1.3.0.rb

```diff
@@ -0,0 +1,40 @@
+# typed: false
+# frozen_string_literal: true
+
+# This file was generated by GoReleaser. DO NOT EDIT.
+class AstroAT130 < Formula
+    desc "To build and run Airflow DAGs locally via docker-compose and deploy DAGs to Astronomer-managed Airflow clusters and interact with the Astronomer API."
+    homepage "https://astronomer.io"
+    version "1.3.0"
+  
+    on_macos do
+      if Hardware::CPU.arm?
+        url "https://github.com/astronomer/astro-cli/releases/download/v1.3.0/astro_1.3.0_darwin_arm64.tar.gz"
+        sha256 "a273704fd300d1e932001bf9a016b4f131cd857c2c8fe195ea92c85aceb6ab96"
+      end
+      if Hardware::CPU.intel?
+        url "https://github.com/astronomer/astro-cli/releases/download/v1.3.0/astro_1.3.0_darwin_amd64.tar.gz"
+        sha256 "26bfac0dce4cd897620bfd8f35d654e7dbab91dfe5eb95f43289aa0d06852bdc"
+      end
+    end
+  
+    on_linux do
+      if Hardware::CPU.arm? && Hardware::CPU.is_64_bit?
+        url "https://github.com/astronomer/astro-cli/releases/download/v1.3.0/astro_1.3.0_linux_arm64.tar.gz"
+        sha256 "cc92bcdd8f446a445590b6495532d61e2bc7c690792f5d3d0b500410afd27515"
+      end
+      if Hardware::CPU.intel?
+        url "https://github.com/astronomer/astro-cli/releases/download/v1.3.0/astro_1.3.0_linux_amd64.tar.gz"
+        sha256 "df4dbe4b08545718f6f720a14c8bcf8e0416518a0dbf3d807529b6fbaae3bd61"  
+      end
+    end
+  
+    def install
+      bin.install "astro"
+    end
+  
+  
+    test do
+      system "#{bin}/astro version"
+    end
+  end
```

Comment:
is it automatic on manual change?
```suggestion
  end

```

Reply:
must have been accidentally updated when verifying the file contents with the checksum file of the release